### DISCUSSION TITLE
Calling "country_name" property on a "App\Entity\Customer" object is not allowed

### DISCUSSION THREAD

**Discussion by chrisonline** [kimai/kimai#5194] [Questions & Answers] [answered]
### Describe the issue

If I want to create an invoice (or preview) I get the following error:
Calling "country_name" property on a "App\Entity\Customer" object is not allowed

![sleekshot](https://github.com/user-attachments/assets/3f36f810-c53f-49d5-8f92-671781364722)

Using latest Kimai version with Docker image.
PHP version 8.3.14

### I already tried

- [X] I've read and searched [the documentation](https://www.kimai.org/documentation/).
- [X] I've searched for similar issues in this repository.
- [X] I've searched for similar issues in [the discussions](https://github.com/kimai/kimai/discussions).

### Kimai version

2.25.0

### How do you run Kimai?

Docker

### Which PHP version are you using?

Other (please mention below)

### Logfile

There is no log in the directory and the "Doctor -> Logs" shows "missing logfile".

![sleekshot](https://github.com/user-attachments/assets/e2045efa-917d-49b9-ac67-ae9982a63308)

**Comment by kevinpapst** [comment, marked as answer]
Custom invoice template I guess. Fix it: `{{ customer.country|country_name }}`

**Reply by chrisonline** [reply]
Correct, I use a custom invoice template and have fixed it now.
Thanks again for your awesome support!
